aries-site/src/examples/components/tag/TagAnatomy.js

```diff
@@ -0,0 +1,140 @@
+import React, { useContext } from 'react';
+import PropTypes from 'prop-types';
+import { Box, Button, Grid, Diagram, Stack, Text, ThemeContext } from 'grommet';
+import { FormClose } from 'grommet-icons';
+import { Annotation } from '../../../layouts';
+
+const color = 'border';
+const anchor = 'vertical';
+const thickness = 'hair';
+const type = 'direct';
+
+const connections = [
+  {
+    anchor,
+    type,
+    color,
+    thickness,
+    fromTarget: 'border-annotation',
+    toTarget: 'border',
+  },
+  {
+    anchor: 'horizontal',
+    type,
+    color,
+    thickness,
+    fromTarget: 'name-annotation',
+    toTarget: 'tagName-2',
+  },
+  {
+    anchor,
+    type,
+    color,
+    thickness,
+    fromTarget: 'value-annotation',
+    toTarget: 'value',
+  },
+  {
+    anchor: 'horizontal',
+    type,
+    color,
+    thickness,
+    fromTarget: 'close-annotation',
+    toTarget: 'close',
+  },
+];
+
+export const AnatomyGrid = ({ ...rest }) => (
+  <Grid
+    columns={['xsmall', 'small', 'xsmall']}
+    gap={{ column: 'small', row: 'medium' }}
+    justify="center"
+    {...rest}
+  />
+);
+
+export const AnatomyBox = ({ id, nameId, valueId, closeId }) => {
```

Comment:
Seems like we have excess height in this anatomy box. 

Looks like we adjusted the plainStyle in StyledButtonKind.js -- review the history and comment related to extra space https://github.com/grommet/grommet/blob/4eb84fa581b14bbae2e75c3e5adfb4f7dd871354/src/js/components/Button/StyledButtonKind.js#L207

![Screen Shot 2022-01-10 at 3 13 03 PM](https://user-images.githubusercontent.com/1756948/148847347-359661b0-66cb-4a17-be2f-eaba9050dc41.png)

Reply:
Hmmm it is seeming like that commit has introduced a variety of alignment issues. We are seeing the effects because I update yarn.lock to latest on grommet stable. Thinking about order of operations in terms of moving forward with this work / handling the regression in ButtonKind separately.

Reply:
We can move forward with this work, just need to make note of the issue on Grommet.

Reply:
I have filed this issue https://github.com/grommet/grommet/issues/5884